pentaho-i18n-bundle/src/test/java/org/pentaho/osgi/i18n/impl/PentahoWebPackageLocalizationServiceTest.java

```diff
@@ -0,0 +1,139 @@
+/*
+ * This program is free software; you can redistribute it and/or modify it under the
+ * terms of the GNU Lesser General Public License, version 2.1 as published by the Free Software
+ * Foundation.
+ *
+ * You should have received a copy of the GNU Lesser General Public License along with this
+ * program; if not, you can obtain a copy at http://www.gnu.org/licenses/old-licenses/lgpl-2.1.html
+ * or from the Free Software Foundation, Inc.,
+ * 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details.
+ *
+ * Copyright 2017 Hitachi Vantara. All rights reserved.
+ */
+
+package org.pentaho.osgi.i18n.impl;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.pentaho.osgi.i18n.LocalizationService;
+import org.pentaho.webpackage.core.PentahoWebPackageResource;
+import org.pentaho.webpackage.core.PentahoWebPackageService;
+
+import java.util.Locale;
+import java.util.ResourceBundle;
+
+import static org.junit.Assert.assertNull;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.eq;
+import static org.mockito.Mockito.any;
+import static org.junit.Assert.assertEquals;
+
+public class PentahoWebPackageLocalizationServiceTest {
+  private static final String NAME_VERSIONED = "test_1.0";
+
+  private PentahoWebPackageLocalizationService pwpLocalizationService;
+  private ClassLoader loaderMock;
+
+  @Before
+  public void setup() {
+    this.pwpLocalizationService = new PentahoWebPackageLocalizationService();
+    this.loaderMock = mock( ClassLoader.class );
+  }
+
+  @Test
+  public void testGetResourceBundleNoWebPackageResource() {
+    setupGetResourceBundleTest();
+
+    ResourceBundle actualResourceBundle = this.pwpLocalizationService
+        .getResourceBundle( getModuleID( "/path/to/resource" ), null );
+
+    assertNull( actualResourceBundle );
+  }
+
+  @Test
+  public void testGetResourceBundleNoResourceBundle() {
+    String resourcePath = "/path/to/resource";
+    Locale locale = Locale.getDefault();
```

Comment:
Why not use a fixed Locale? getDefault() varies form computer to computer and that is not something that we want to test in this test.